js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
```

Comment:
Why are we exporting them? Aren't we making them global?

Reply:
I am declaring these variables on addToTask file and adding value at that file only. after that I am importing id and LIST to other files too so that I can update properties of LIST